packages/forma-36-react-components/src/components/EmptyState/EmptyState.css

```diff
@@ -0,0 +1,29 @@
+@import 'settings/colors';
+@import 'settings/typography';
+@import 'settings/transitions';
+
+.EmptyState {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+}
+.EmptyState_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.EmptyState_illustration {
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.EmptyState_element {
+  max-width: 500px;
```

Comment:
We don't have any nice helpers, and this is a good reminder to add documentation - we try to use REMs for all spacing so that if a user has their browser font size set to a different size, the UI will resize to accommodate.

We've been using `calc` to help us write these in a way that you can see the original desired pixel value.

tl;dr: please change to `max-width: calc(1rem * (500 / var(--font-base-default)));`